### ISSUE TITLE
See a few broken links and typos in the readme file and tutorial

### ISSUE THREAD

**Issue by petec029** [blockstack/blockstack-ios#65]
1) At the top of the readme.md file it says Android.

This repository contains a pre-release for Android developers:
the Blockstack Android SDK (/blockstack-sdk)

2) The link /blockstack-sdk is broken

3) In step 6 of the tutorial is says 'Navigate to https://localhost:8080 with your browser to display the application.' while the previous step shows that the 'Project is running at http://localhost:8080/'

So step 6 should be http not https

4) The link to download the final project code for the tutorial does not do anything.  Just reloads the page.

If you prefer, you can skip working through the tutorial all together. Instead, you can download the final project code and import it into XCode to review it.

**Comment by moxiegirl** [comment]
Thanks for the issue @petec029 I'm working on this and also building the zip. Apologies for the time frame, I couldn't find the original ZIP so I am needing to recreate it

**Comment by larrysalibra** [comment]
I still see the language about Android in the readme.

**Comment by larrysalibra** [comment]
The link @petec029 mentioned is also still broken.

**Comment by moxiegirl** [comment]
@larrysalibra The problems **were** fixed in the served documentation which you can see by the merged PR refrenced  above (https://github.com/blockstack/docs.blockstack/pull/23 and https://github.com/blockstack/docs.blockstack/pull/24.)  and which users can find here:

https://docs.blockstack.org/ios/tutorial.html

Unfortunately, yes, you are correct, I forgot about the `ios` repo's README.  I also did not port the change into the `docs/` source here. A bit back, I had switched the way I was doing docs source --- my changes were not being pulled fast enough and I had no perms to merge my own requests on the `ios` repo.  Which meant relying on the source here as the sole docs source slowed up the doc development.  Aaron changed the perms beginning of this month so I'm hoping to redo a lot of my production at the end. 

In short :-) I forgot about this end but it was fixed on the docs site when  @petec029  brought it up.  I'll do port today and tweak the README.

**Comment by moxiegirl** [comment]
Closing this as it was merged. Reopen if you are still seeing issues.
